kotlin-js/src/jsMain/kotlin/js/typedarrays/Uint8Array.kt

```diff
@@ -18,5 +22,12 @@ open external class Uint8Array(
     constructor(elements: JsIterable<Byte>)
     constructor(elements: ReadonlyArray<Byte>)
 
+    inline fun asByteArray(): ByteArray =
```

Comment:
`to`?

Reply:
@turansky yeah that was my original idea. However, maybe better to keep it the same as for `Int8Array`?

Reply:
There is no conversion in `Int8Array.asByteArray`

Reply:
Renamed to `toByteArray`.